app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/CharacterFragment.kt

```diff
@@ -0,0 +1,62 @@
+package com.puzzlebench.clean_marvel_kotlin.presentation
+
+import android.app.Dialog
+import android.app.DialogFragment
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.puzzlebench.clean_marvel_kotlin.data.service.CharacterServicesImpl
+import com.puzzlebench.clean_marvel_kotlin.domain.usecase.GetCharacterByIdUseCase
+import com.puzzlebench.clean_marvel_kotlin.presentation.mvp.CharacterFragmentPresenter
+import com.puzzlebench.clean_marvel_kotlin.presentation.mvp.CharecterFragmentView
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+/**
+ * A simple [Fragment] subclass.
+ * Activities that contain this fragment must implement the
+ * [CharacterFragment.OnFragmentInteractionListener] interface
+ * to handle interaction events.
+ * Use the [CharacterFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ *
+ */
+class CharacterFragment : DialogFragment() {
+
+    companion object {
+        private val CHARACTER_ID = "character_id"
+
+        fun newInstance(character_id: Int): CharacterFragment {
+            val args = Bundle()
+            args.putInt(CHARACTER_ID, character_id)
+            val fragment = CharacterFragment()
+            fragment.arguments = args
+            return fragment
+        }
+    }
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        val view = super.onCreateDialog(savedInstanceState)
```

Comment:
you can use oneLineFunctions here 

override fun onCreateDialog(savedInstanceState: Bundle?): Dialog = super.onCreateDialog(savedInstanceState)

Reply:
Done! I remove because don´t use! Thnk´s